contracts/StaderConfig.sol

```diff
@@ -537,6 +558,11 @@ contract StaderConfig is IStaderConfig, AccessControlUpgradeable {
         return hasRole(OPERATOR, account);
     }
 
+    function isAllowedToCall(address account, string calldata functionSig) external view returns (bool) {
```

Comment:
Lets add natspec comments as well, so that other devs are aware that this view function is useful only when called by specific contracts.
------
Shall we also expose one more view function with `contractAddress` as well ? this would helpful to manually check using etherscan ?